resources/js/components/Graphql.vue

```diff
@@ -103,9 +103,12 @@ export default {
                 if (this.cache) {
                     useLocalStorage(this.cachePrefix + this.cache, null, { serializer: StorageSerializers.object }).value = this.data
                 }
+
+                return this.data
             } catch (error) {
                 console.error(error)
                 this.errorCallback(this.dataVariables, await error?.response?.json())
+                return false
             } finally {
                 this.running = false
```

Comment:
Does this still work? I can imagine that a `finally` doesn't run when you return prematurely.